### ISSUE TITLE
TF Lite API Reference on Internet Page does not match Code for interpreter_options.h

### ISSUE THREAD

**Issue by PfeifferJosef-96** [tensorflow/tensorflow#57208]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Documentation Bug

### Source

source

### Tensorflow Version

Master Branch

### Custom Code

No

### Current Behaviour?

```shell
The tensorflow API reference for tf lite is showing a different name for a method for interpreter_options.h (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/interpreter_options.h).

The webpage shows the name "SetDynamicAllocationForLargeTensors" [https://www.tensorflow.org/lite/api_docs/cc/class/tflite/interpreter-options#setdynamicallocationforlargetensors] but the c++ code has the name "OptimizeMemoryForLargeTensors".

I expect that both names are equal. Either the on or the other.
```

### Standalone code to reproduce the issue

```shell
N/A
```

### Relevant log output

_No response_</details>

**Comment by mohantym** [comment]
Hi @PfeifferJosef-96 !
Thanks for pointing it out.

@sachinprasadhs !
Could you please look at this issue.

Thank you!

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue, I have created the fix internally, on successful merge it should close the issue.

**Comment by sachinprasadhs** [comment]
I see that the document has to be autoupdated from the code file, for some reason it is not synced, since the Tensorflow 2.10 is around the corner, that should fix this issue. Let's wait for a while.

**Comment by markmcd** [comment]
The docs reflect what's in the *released* version of TensorFlow, which is v2.9.1.

If you are looking for the matching source code, clone the [v2.9.1 tag](https://github.com/tensorflow/tensorflow/tree/v2.9.1/tensorflow/lite).

**Comment by PfeifferJosef-96** [comment]
Thank you all for your work!

I think this issue can be closed then.
